src/test/java/com/project/playlist/service/impl/PlaylistServiceTest.java

```diff
@@ -0,0 +1,144 @@
+package com.project.playlist.service.impl;
+
+import com.project.playlist.exceptions.PlaylistNotFoundException;
+import com.project.playlist.exceptions.UserNotFoundException;
+import com.project.playlist.model.Playlist;
+import com.project.playlist.model.User;
+import com.project.playlist.repository.PlaylistRepository;
+import com.project.playlist.service.PlaylistService;
+import com.project.playlist.service.UserService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.function.Executable;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.ArrayList;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.Mockito.*;
+
+@SpringBootTest
+@ExtendWith(MockitoExtension.class)
+public class PlaylistServiceTest {
+    private PlaylistService playlistService;
+    @Mock
+    private PlaylistRepository playlistRepositoryMock;
+    @Mock
+    private UserService userServiceMock;
+
+    private User user;
+    private Playlist actualPlaylist;
+    private Playlist expectedEmptyPlaylist;
+    private Playlist emptyPlaylistToSave;
+    private final Long invalidUserId = 111L;
+    private final Long invalidPlaylistId = 111L;
+
+
+    @BeforeEach
+    void setUp() {
+        playlistService = new PlaylistServiceImpl(playlistRepositoryMock, userServiceMock);
+        user = User.builder()
+                .id(1L)
+                .username("username")
+                .email("email")
+                .password("pass")
+                .playlists(new ArrayList<>())
+                .videos(new ArrayList<>())
+                .build();
+        emptyPlaylistToSave = Playlist.builder()
+                .name("playlistName")
+                .user(user)
+                .playlistVideos(new ArrayList<>())
+                .build();
+        expectedEmptyPlaylist = Playlist.builder()
+                .id(1L)
+                .name("playlistName")
+                .user(user)
+                .playlistVideos(new ArrayList<>())
+                .build();
+        lenient().when(userServiceMock.getUserById(eq(1L))).thenReturn(user);
+    }
+
+    @Test
+    void givenUserAndPlaylistName_whenCreateEmptyPlaylist_thenReturnCreatedPlaylist() {
+        //given
+        lenient().when(playlistRepositoryMock.save(emptyPlaylistToSave)).thenReturn(expectedEmptyPlaylist);
+        //when
+        actualPlaylist = playlistService.createEmptyPlaylist(1L, "playlistName");
+        //then
+        verifyAll(1, 1, 0, 0);
+        assertEquals(expectedEmptyPlaylist, actualPlaylist);
+    }
+
+    @Test
+    void givenNonexistentUserAndPlaylistName_whenCreateEmptyPlaylist_thenThrowException() {
```

Comment:
Add exception type in test name like:
`givenNonexistentUserAndPlaylistName_whenCreateEmptyPlaylist_thenThrowUserNotFoundException()`